templates/taskcat-verify-all.yaml

```diff
@@ -56,27 +61,25 @@ Resources:
     Properties:
       Parameters:
         LaunchRoleName: ''
-        CreateEndUsers: 'No'
-        LinkedRole1: !Ref 'LinkedRole1'
-        LinkedRole2: !Ref 'LinkedRole2'
+        PrincipalRoleName1: ''
         RepoRootURL: !Ref 'RepoRootURL'
         ProductVersionName: !Ref 'ProductVersionName'
       TemplateURL: !Sub '${RepoRootURL}ec2/sc-portfolio-ec2.yaml'
       TimeoutInMinutes: 5
-  STACKvpcportfolio:
-    Type: AWS::CloudFormation::Stack
-    DependsOn:
-      - IAMenduser
-    Properties:
-      Parameters:
-        LaunchRoleName: !GetAtt 'STACKec2portfolio.Outputs.LaunchConstraintRoleName'
-        CreateEndUsers: 'No'
-        LinkedRole1: !Ref 'LinkedRole1'
-        LinkedRole2: !Ref 'LinkedRole2'
-        RepoRootURL: !Ref 'RepoRootURL'
-        ProductVersionName: !Ref 'ProductVersionName'
-      TemplateURL: !Sub '${RepoRootURL}vpc/sc-portfolio-vpc.yaml'
-      TimeoutInMinutes: 5
+  # STACKvpcportfolio:
```

Comment:
why not delete this?  if there's a good reason can we add a comment describing the reason for keeping it as comments?

Reply:
done